reactive/kubernetes_master.py

```diff
@@ -326,6 +326,20 @@ def install_snaps():
     remove_state('kubernetes-master.components.started')
 
 
+@when('etcd.available')
+@when('config.changed.enable-metrics')
+def enable_metric_changed():
```

Comment:
Do we want to add a @when('kubernetes-master.components.started') here? If we get config changed before things are started we will attempt to configure the apiserver and cdk-addons before they are ready. I'm not sure if this would cause trouble, but could it be before the snap is installed? On the other hand, if we ignore a config changed if the components aren't started, starting the components pretty much runs this function, so it will start correctly the first time, right?